wireapp/wire-android-reloaded
# feat: Group call view - Part 3 : order participants(AR-1148)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1148" title="AR-1148" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-1148</a>  View all call participants during the call (grid)
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Description

A follow up PR of #700 

Ordering participants is done in kalium side, this PR is a fix of the breaking changes made there

### Dependencies (Optional)

Blocked by:
[feat(calling): sort participants alphabetically (AR-1148)
](https://github.com/wireapp/kalium/pull/624)
Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

Start a group call a check if the grid is well sorted
The self user is always shown first in the grid

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
